Classes/URKArchive.mm

```diff
@@ -331,7 +403,32 @@ - (nullable NSURL *)firstVolumeURL {
 
 - (nullable NSArray<NSString*> *)listVolumePaths:(NSError **)error
 {
-    return @[];
+    __block NSMutableArray<NSString*> *volumePaths = [NSMutableArray new];
```

Comment:
This doesn’t need to be marked as a block variable if you’re not setting it from within a block. Even adding members to a mutable collection can be done without __block.